ReSharper.FSharp/src/FSharp.Psi.Intentions/src/QuickFixes/UseNestedRecordFieldSyntaxFix.fs

```diff
@@ -0,0 +1,45 @@
+namespace JetBrains.ReSharper.Plugins.FSharp.Psi.Features.Daemon.QuickFixes
+
+open System.Collections.Generic
+open JetBrains.ReSharper.Plugins.FSharp.Psi
+open JetBrains.ReSharper.Plugins.FSharp.Psi.Features.Daemon.Highlightings
+open JetBrains.ReSharper.Plugins.FSharp.Psi.Features.Refactorings
+open JetBrains.ReSharper.Plugins.FSharp.Psi.Impl
+open JetBrains.ReSharper.Plugins.FSharp.Psi.Impl.Tree
+open JetBrains.ReSharper.Plugins.FSharp.Psi.Tree
+open JetBrains.ReSharper.Psi.ExtensionsAPI
+open JetBrains.ReSharper.Resources.Shell
+
+type UseNestedRecordFieldSyntaxFix(warning: NestedWithCanBeSimplifiedWarning) =
+    inherit FSharpScopedQuickFixBase(warning.Binding)
+
+    let bindingExpr = warning.Binding
```

Comment:
Is it a binding or an expression? The rhs expression implies the first, while the name implies the second.